infrastructure/main.tf

```diff
@@ -108,7 +108,7 @@ module "recipe-database" {
   subscription       = var.subscription
 }
 
-module "recipe-database-v11" { 
+module "recipe-database-v11" {
```

Comment:
isn't this removed?

Reply:
Darren was working on this repo by mistake and had created a branch with the same name. Will close this PR as it is tainted.